spec/integration_spec.cr

```diff
@@ -6,9 +6,9 @@ describe LuckyRouter do
 
     # Here to makes sure things run super fast even with lots of routes
     1000.times do
-      router.add("put", "#{(rand * 100).to_i}", :fake_show)
-      router.add("get", "#{(rand * 100).to_i}/edit", :fake_edit)
-      router.add("get", "#{(rand * 100).to_i}/new/edit", :fake_new_edit)
+      router.add("put", "#{UUID.random}", :fake_show)
+      router.add("get", "#{UUID.random}/edit", :fake_edit)
+      router.add("get", "#{UUID.random}/new/edit", :fake_new_edit)
```

Comment:
![image](https://user-images.githubusercontent.com/17329408/98057992-42409600-1e09-11eb-89b7-0cad9808d8a8.png)
It needs more random.